contracts/HubPool.sol

```diff
@@ -59,25 +67,41 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
 
     mapping(uint256 => CrossChainContract) public crossChainContracts; // Mapping of chainId to the associated adapter and spokePool contracts.
 
+    LpTokenFactoryInterface lpTokenFactory;
+
     FinderInterface public finder;
 
-    bytes32 public identifier;
+    // When bundles are disputed a price request is enqueued with the DVM to resolve the resolution.
+    bytes32 public identifier = "IS_ACROSS_V2_BUNDLE_VALID";
```

Comment:
I decided to seed the `identifier` and `lpFeeRatePerSecond` so they dont need to be passed in via the constructor. can always be changed later.